rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentAgent.scala

```diff
@@ -162,14 +157,50 @@ final class AsyncDeploymentAgent(
     )
   }
 
-  override protected def lowPriority = {
+  // That flag file is used to determine if a policy update was running when the webapp stopped so we can launch a new one when starting
+  val policyUpdateRunningFlagPath = "/opt/rudder/etc/policy-update-running"
```

Comment:
I agree it's a bit ugly to have a state file under etc, but you can look at it the other way round, and see it as a flag to signal to the webapp it has to do something. Linux uses a similar mechanism during shutdown, when in touches /etc/nologin to prevent sshd/telnetd/etc to allow new users to login - it is both a config file and a state file, from different perspectives.